packages/suite/src/views/settings/device/DeviceLabel.tsx

```diff
@@ -25,14 +26,33 @@ export const DeviceLabel = ({ isDeviceLocked }: DeviceLabelProps) => {
         applySettings: deviceSettingsActions.applySettings,
     });
     const { anchorRef, shouldHighlight } = useAnchor(SettingsAnchor.DeviceLabel);
+    const { translationString } = useTranslation();
```

Comment:
You can store just an error key in `error` state and use translation component with value `length: 16` even though the string is different.

Reply:
I saw it used like this in other onChange functions so I would keep it like that